src/Expectation.php

```diff
@@ -165,6 +166,10 @@ public function sequence(...$callbacks): Expectation
             $index       = $index < count($values) - 1 ? $index + 1 : 0;
         }
 
+        if ($callbacksCount > count($values)) {
```

Comment:
The reason I added this condition:

https://github.com/pestphp/pest/blob/2f0cd7a4e3a36ac51367ea45d65fc09403ec32c7/tests/Features/Expect/sequence.php#L18-L27